tests/derive/test_at_university_coding.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import yaml
+from at_university_testcases import test_data
+from chispa import assert_df_equality
+
+from cishouseholds.derive import assign_regex_match_result
+from cishouseholds.regex_patterns import at_university_pattern
+
+
+@pytest.fixture
+def at_university_test_cases():
```

Comment:
This is generic, so might be worth refactoring into a function that gets used for all tests